src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/gerritnotifier/ParameterExpander.java

```diff
@@ -242,13 +282,31 @@ private Integer getBuildStartedCodeReviewValue(Run r) {
         if (r != null) {
             map.put("BUILDURL", jenkins.getRootUrl() + r.getUrl());
         }
-        map.put("VERIFIED", String.valueOf(verified));
-        map.put("CODE_REVIEW", String.valueOf(codeReview));
+        map.put(VERIFIED, String.valueOf(verified));
+        map.put(CODE_REVIEW, String.valueOf(codeReview));
         map.put("NOTIFICATION_LEVEL", notifyLevel);
 
         return map;
     }
 
+    private Map<String, String> createStandardParameters(Run r, GerritTriggeredEvent gerritEvent, Map<String, Integer> labelVotes, String notifyLevel) {
+        int codeReviewValue = 0;
+        int verifiedValue = 0;
+
+        if(labelVotes.containsKey(CODE_REVIEW))
+            codeReviewValue = labelVotes.get(CODE_REVIEW);
```

Comment:
@GeorgeCimpoies Need to handle null values being put here. Or, just make sure we don't put `null` in the first place 🤔 ... Maybe still better to check here for null instead, since we can't assume we cover all external cases (or just use Optional) :)